sorc/checkout.sh

```diff
@@ -156,7 +156,7 @@ mkdir -p "${logdir}"
 # The checkout version should always be a speciifc commit (hash or tag), not a branch
 errs=0
 checkout "gfs_utils.fd"    "https://github.com/NOAA-EMC/gfs-utils"              "0b8ff56"                    ; errs=$((errs + $?))
-checkout "ufs_model.fd"    "https://github.com/ufs-community/ufs-weather-model" "${ufs_model_hash:-7a1ce44}" ; errs=$((errs + $?))
+checkout "ufs_model.fd"    "https://github.com/ufs-community/ufs-weather-model" "${ufs_model_hash:-ce86979}" ; errs=$((errs + $?))
```

Comment:
Also update `Externals.cfg`. For now, we're still maintaining both.

Reply:
ok

Reply:
done.